datetime/day_of_year.ts

```diff
@@ -13,14 +13,39 @@ import { DAY } from "./constants.ts";
  * dayOfYear(new Date("2019-03-11T03:24:00")); // output: 70
  * ```
  *
- * @return Number of the day in year
+ * @return Number of the day in the year in the local time zone
  */
 export function dayOfYear(date: Date): number {
   // Values from 0 to 99 map to the years 1900 to 1999. All other values are the actual year. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date)
   // Using setFullYear as a workaround
 
   const yearStart = new Date(date);
 
+  yearStart.setFullYear(date.getFullYear(), 0, 0);
+  const diff = date.getTime() -
+    yearStart.getTime();
+
+  return Math.floor(diff / DAY);
+}
+
+/**
+ * Returns the number of the day in the year in UTC time.
+ *
+ * @example
+ * ```ts
+ * import { dayOfUtcYear } from "https://deno.land/std@$STD_VERSION/datetime/mod.ts";
+ *
+ * dayOfUtcYear(new Date("2019-03-11T03:24:00.000Z")) // output 70
+ * ```
+ *
+ * @return Number of the day in the year in UTC time
+ */
+export function dayOfUtcYear(date: Date): number {
```

Comment:
`Utc` in the middle of `dayOfYear` sounds strange to me. How about `utcDayOfYear` or `dayOfYearUtc`?

Reply:
I prefer `dayOfYearUtc`. Updated.